src/vos/tests/vts_aggregate.c

```diff
@@ -2982,7 +2982,7 @@ run_discard_tests(const char *cfg)
 {
 	char	test_name[DTS_CFG_MAX];
 
-	dts_create_config(test_name, "VOS Discard Tests %s", cfg);
+	dts_create_config(test_name, "Discard Tests %s", cfg);
```

Comment:
So the log name was generated by the first word of test name? This patch adjusted the test name so that we'll be able to have separated logs for each VOS test?

Reply:
This is more for categorization.  If you look at test results from classic view in Jenkins, you will see the main vos category and now under that you'll see the subcategories with the tests under them (e.g. Aggregation) Before, the categories came from the first word of the test suite.  If you click on the results here, I think you'll find it cleaner than it was previously.